pkg/controller/applicationlayer/applicationlayer_controller.go

```diff
@@ -105,8 +107,21 @@ func add(mgr manager.Manager, c controller.Controller) error {
 		return fmt.Errorf("applicationlayer-controller failed to watch Tigera network resource: %v", err)
 	}
 
-	// Watch configmaps created for envoy.
-	for _, configMapName := range []string{applicationlayer.EnvoyConfigMapName} {
+	// Watch for configmap changes in tigera-operator namespace; the cm contains ruleset for ModSecurity library:
+	err = utils.AddConfigMapWatch(c, applicationlayer.ModSecurityRulesetConfigMapName, common.OperatorNamespace())
```

Comment:
Looks like we're adding a watch for this twice? We should only do it once.

Reply:
yep that makes total sense, I was just reading too quickly and missed the nuance.